addon/templates/components/cfb-form-editor/question/default.hbs

```diff
@@ -0,0 +1,11 @@
+{{component labelComponent}}
+
+{{component (get-widget this.field.question)
+  field=this.field
+  disabled=false
+  context=(hash)
```

Comment:
Can't this be deleted?

Reply:
Probably, yes. I'll try and remove it if possible.

Reply:
Seems to work without it :sunglasses: